openmc/data/data.py

```diff
@@ -3,89 +3,89 @@
 import re
 
 
-# Isotopic abundances from M. Berglund and M. E. Wieser, "Isotopic compositions
-# of the elements 2009 (IUPAC Technical Report)", Pure. Appl. Chem. 83 (2),
-# pp. 397--410 (2011).
+# Isotopic abundances from Meija J, Coplen T B, et al, "Isotopic compositions
+# of the elements 2013 (IUPAC Technical Report)", Pure. Appl. Chem. 88 (3),
+# pp. 293-306 (2013).
```

Comment:
Can you add a comment mentioning the columns from the paper, i.e. something like 'The "representative isotopic abundance" values from column 9 are used except where an interval is given, in which case the "best measurement" is used.'